packages/jest-runtime/src/index.ts

```diff
@@ -2372,7 +2371,7 @@ export default class Runtime {
       },
       getSeed: () => {
         // TODO: remove this check in Jest 30
-        if (this._globalConfig?.seed === undefined) {
+        if (this._globalConfig.seed === undefined) {
```

Comment:
Initially I wanted to remove these checks since they are not needed in Jest 30. But here it also makes sure that `this._globalConfig` is defined.